luxtronik/datatypes.py

```diff
@@ -42,7 +42,7 @@ def options(self):
     def from_heatpump(self, value):
         if value in self.codes:
             return self.codes.get(value)
-        return None
+        return value
```

Comment:
This change is unrelated to this pull request. In case it is needed / wanted, I would suggest to have another pull request, as to not mix up pull requests and discussions.

Reply:
Done, see #91

Reply:
Okay, I've rebased the branch in order to not have a "revert" commit that will only confuse history ;-). Thank you for your other issue / PR in #91.